pkg/scaling/scale_handler.go

```diff
@@ -208,7 +208,9 @@ func (h *scaleHandler) checkScaledObjectScalers(ctx context.Context, scalers []s
 			continue
 		} else if isTriggerActive {
 			isActive = true
-			h.logger.V(1).Info("Scaler for scaledObject is active", "Metrics Name", scaler.GetMetricSpecForScaling()[0].External.Metric.Name)
+			if scaler.GetMetricSpecForScaling()[0].External != nil {
```

Comment:
Nit: I'd add Resource Metric Name to the debug message as well. (ie. in case user is debugging and is curious why the particular ScaledObject is active)

Reply:
I agree, thanks review.